herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_array/IX_detector_array.m

```diff
@@ -542,6 +542,20 @@
             end
         end
         
+        function val = get_detpar_representation(obj)
+            %GET_DETPAR convert first detector bank into detpar struct
+            % intended for use initialising from a *default* ix_detector_array
```

Comment:
What is a `default` here? Empty constructor?

Reply:
yes

Reply:
and commented appropriately